modules/dev-tools/backtrace-helper.php

```diff
@@ -0,0 +1,51 @@
+<?php
+namespace Elementor\Modules\DevTools;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Backtrace_Helper {
+	/**
+	 * find_who_called_me
+	 * Retrieves the function, class, file, line, type and name of the function that called the function that called this function.
+	 *
+	 * @param  int $stack_depth The depth of the stack to look for.
+	 * @return array
+	 */
+	public static function find_who_called_me( $stack_depth ) {
+		$backtrace = debug_backtrace( DEBUG_BACKTRACE_IGNORE_ARGS );
+
+		$caller = $backtrace[ $stack_depth ];
```

Comment:
What if $bactrace is not an array or the `$stack_depth` key is not exists in the array?